python/ray/data/impl/shuffle.py

```diff
@@ -52,9 +53,9 @@ def simple_shuffle(
 
     shuffle_map_out = [
         shuffle_map.options(
-            **map_ray_remote_args,
+            **{"name": f"[{dataset_name}]:shuffle_map_{i}", **map_ray_remote_args},
```

Comment:
What's the best way to test this: grep worker-xxx.err?

Reply:
Since it's going to be in the core worker logs, I think grepping those log files is probably your best option. You could redirect everything to stderr and use the `capfd` fixture, as I did [here](https://github.com/ray-project/ray/blob/53c4c7b1be54fe67a54b34f3f0c076ad502e35cc/python/ray/tests/test_logging.py#L279-L353), but I don't know if the test should rely on that feature.